contrib/utilities/indent

```diff
@@ -31,22 +45,26 @@ if test ! -f contrib/utilities/astyle.rc ; then
   exit 1
 fi
 
-if test -z "`which astyle`" ; then
+astyle="$(find_astyle)"
+
+if test -z "${astyle}"; then
   echo "*** No astyle program found."
   exit 1
 fi
 
-if test "`astyle --version 2>&1`" != "Artistic Style Version 2.04" ; then
+if test "$("${astyle}" --version 2>&1)" != "Artistic Style Version 2.04" ; then
```

Comment:
```suggestion
if test "$(${astyle} --version 2>&1)" != "Artistic Style Version 2.04" ; then
```

Reply:
Same as below. This would fail without the extra quotes if there happens to be spaces in `$astyle`.